src/Twig/Extension/PaginationRuntime.php

```diff
@@ -109,20 +109,22 @@ public function filter(
     }
 
     /**
-     * @param array<string, mixed> $query
-     * @param int $page
+     * @param array<string, mixed>      $query
+     * @param int                       $page
+     * @param array<string, mixed>|null $paginatorOptions
      * @return array<string, mixed>
      */
-    public function getQueryParams(array $query, int $page): array
+    public function getQueryParams(array $query, int $page, ?array $paginatorOptions = []): array
```

Comment:
same name change here. Moreover, we don't need to accept a null value

Reply:
Changed as requested